jekyll/_cci2/creating-orbs.md

```diff
@@ -6,39 +6,36 @@ description: "Starting point for Creating CircleCI Orbs"
-`echo '{version: "2.1", description: "a sample orb"}' > /tmp/orb.yml`
+#### Steps 1 - 3 - Use the CircleCI CLI
+The first step you must perform in the orbs publishing process is to download and install, update, and then configure the CircleCI CLI for use with CircleCI orbs. Detailed instructions for how to install, update and configure the CLI can be found in the [Get the new CircleCI CLI](https://circleci.com/docs/2.0/creating-orbs/#get-the-new-circleci-cli) section on this page.
 
-4. Validate that your code is a valid orb using the CLI. For example, using the path above you could use:
-`circleci orb validate /tmp/orb.yml`
+#### Step 4 - Validate your build configuration
+Once you have configured the CircleCI CLI, you then need to validate your build configuration to ensure you can correctly create or update an orb. You can find detailed instructions on how to validate your build configuration in the [Validating a Build Config] (https://circleci.com/docs/2.0/creating-orbs/#validating-a-build-config) section.
 
-5. Publish a development version of your orb, for example:
-`circleci orb publish /tmp/orb.yml sandbox/hello-world@dev:first`
+#### Step 5 - Set the Version Property to 2.1
+After validating your build configuration, set the version property to 2.1. More information on how to set the version property can be found in the [Set the Version Property to 2.1](https://circleci.com/docs/2.0/creating-orbs/#set-the-version-property-to-21) section on this page.
 
-6. Once you are ready to push your orb to production, you can publish it manually using ```circleci orb publish``` command or promote it directly from the development version. To increment the new dev version to become 0.0.1, use the following command:
-`circleci orb publish promote sandbox/hello-world@dev:first patch`
+#### Step 6 - Create a new Orb using inline template
+CircleCI provides you an example inline template that you can, and should, use when creating your own orbs. The example inline teamplat can be found in the [Example Inlline Template](https://circleci.com/docs/2.0/creating-orbs/#example-inline-template) section on this page.
 
-7. Your orb is now published in an immutable form as a production version and can be used safely in builds. You can view the source of your orb by using:
-`circleci orb source sandbox/hello-world@0.0.1`
+#### Step 7
+Now that you have copied the example inline template from step 6, you will need to add your own orb elements (Jobs, Commands, and Executors to this inline template. For more information on these orb elements, refer to the [Commands](https://circleci.com/docs/2.0/using-orbs/#commands), [Jobs](https://circleci.com/docs/2.0/using-orbs/#jobs), and [Executors](https://circleci.com/docs/2.0/using-orbs/#executors) sections found in the [Using Orbs](https://circleci.com/docs/2.0/using-orbs/#section=configuration) page.
```

Comment:
I really don't like `need` and `must` in docs language unless something is actually a required dependency to make something work.

Reply:
Agreed....I am going to remove these from the document

Reply:
Removed "need" and must" from document where appropriate